mkdocs.yml

```diff
@@ -34,8 +34,7 @@ nav:
     - Getting Started: 'develop/index.md'
     - Install Locally: 'develop/install_locally.md'
     - Run: 'develop/run.md'
-    - Configuration: 'develop/config.md'
-    - Customization: 'develop/customize.md'
+    - Make it your own: 'develop/make_it_your_own.md'
```

Comment:
Shelved:  For the future I think we will need something like

``` yaml
- Make it your own: index.yaml
  - Change the logo: logo.yaml
  - Custom vocabularies: vocabs.yaml
  - Metadata extensions: metaext.yaml
  [...]
```

Naming just a bit random, my point is: What do you think of this structure? Just not to have a super long page.

Reply:
Yes I agree that we might want to break those up in the future: some might be short though.